build/makelib/helm.mk

```diff
@@ -38,6 +38,11 @@ $(HELM):
 define helm.chart
 $(HELM_OUTPUT_DIR)/$(1)-$(VERSION).tgz: $(HELM) $(HELM_OUTPUT_DIR) $(shell find $(HELM_CHARTS_DIR)/$(1) -type f)
 	@echo === helm package $(1)
+	@if [ ! -f $(HELM_CHARTS_DIR)/$(1)/charts/ceph-csi-operator-*.tgz ]; then \
+		echo "=== helm repo add (dependencies missing)"; \
```

Comment:
Is the condition on line 41 working for you? I just tried building locally and every time it printed this line that it has dependencies missing.

Reply:
did you remove other tar library tar file as well? I'll check this again

Reply:
As discussed, we will follow up separately on this issue